mne_bids/tests/test_read.py

```diff
@@ -693,7 +693,8 @@ def test_handle_eeg_coords_reading(tmp_path):
     montage = mne.channels.make_dig_montage(ch_pos=ch_pos,
                                             coord_frame="unknown")
     raw.set_montage(montage)
-    with pytest.warns(RuntimeWarning, match="Skipping EEG electrodes.tsv"):
+    with pytest.raises(RuntimeError, match="'head' coordinate frame "
```

Comment:
Wait, what? So this is a breaking behavior change? If yes, we might need a deprecation cycle for this?

Reply:
I don't think there is anything to cycle, it's raising an error where before there was a warning. I think this is appropriate and needed; I really don't like when among all the things that are printed out you get a warning that the channels weren't written which is kind of a big deal if you think they are supposed to be. This is a pretty edge case anyway, when you have a montage in `head` with no fiducials.